nuxeo-dm/nuxeo-platform-default-config/src/main/resources/OSGI-INF/default-cache-contrib.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.default.caches">
+
+  <extension target="org.nuxeo.ecm.core.cache.CacheService"
+             point="caches">
```

Comment:
Not sure what the code style here, I'm used to put everything on the same line if it's not longer than 120 char. Wdyt ?